File: android/app/src/main/kotlin/com/sopro/sopro/logging/LogLevel.kt
package com.sopro.sopro.logging

// Níveis de severidade. Ordem crescente: TRACE < DEBUG < INFO < WARN < ERROR < FATAL.
// ordinal do enum reflete a ordem — usado em shouldEmit() do Logger.
enum class LogLevel(val label: String) {
    TRACE("TRACE"),
    DEBUG("DEBUG"),
    INFO("INFO"),
    WARN("WARN"),
    ERROR("ERROR"),
    FATAL("FATAL"),
}

File: lib/infrastructure/logging/core/log_level.dart
// Níveis de severidade disponíveis no Logger.
// Ordem crescente: trace < debug < info < warn < error < fatal.
enum LogLevel {
  trace,
  debug,
  info,
  warn,
  error,
  fatal;

  // Rótulo em maiúsculas para serialização e exibição.
  String get label => name.toUpperCase();
}
